demos/titanic/demo.ipynb

```diff
@@ -271,45 +285,58 @@
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
        "      <th>model name</th>\n",
+       "      <th>n features</th>\n",
        "      <th>holdout error</th>\n",
+       "      <th>holdout error interval</th>\n",
        "      <th>cv error</th>\n",
        "      <th>training error</th>\n",
        "    </tr>\n",
        "  </thead>\n",
        "  <tbody>\n",
        "    <tr>\n",
-       "      <th>0</th>\n",
-       "      <td>xg_boost</td>\n",
-       "      <td>-0.8101</td>\n",
-       "      <td>-0.8042</td>\n",
-       "      <td>-0.8822</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
        "      <th>1</th>\n",
        "      <td>random_forest</td>\n",
-       "      <td>-0.8101</td>\n",
-       "      <td>-0.8462</td>\n",
-       "      <td>-0.7698</td>\n",
+       "      <td>9</td>\n",
+       "      <td>-82.68</td>\n",
+       "      <td>-86.59, -79.89</td>\n",
+       "      <td>-79.72</td>\n",
+       "      <td>-88.58</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>xg_boost</td>\n",
+       "      <td>1</td>\n",
+       "      <td>-79.89</td>\n",
+       "      <td>-83.8, -75.98</td>\n",
+       "      <td>-76.92</td>\n",
+       "      <td>-78.91</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>2</th>\n",
        "      <td>linear_model</td>\n",
-       "      <td>-0.7765</td>\n",
-       "      <td>-0.8671</td>\n",
-       "      <td>-0.8014</td>\n",
+       "      <td>15</td>\n",
+       "      <td>-70.39</td>\n",
+       "      <td>-74.86, -65.36</td>\n",
+       "      <td>-76.22</td>\n",
+       "      <td>-69.6</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "      model name holdout error cv error training error\n",
-       "0       xg_boost       -0.8101  -0.8042        -0.8822\n",
-       "1  random_forest       -0.8101  -0.8462        -0.7698\n",
-       "2   linear_model       -0.7765  -0.8671        -0.8014"
+       "      model name  n features holdout error holdout error interval cv error  \\\n",
+       "1  random_forest           9        -82.68         -86.59, -79.89   -79.72   \n",
+       "0       xg_boost           1        -79.89          -83.8, -75.98   -76.92   \n",
```

Comment:
I'm confused by this one:

model | name | n features | holdout error | holdout error interval | cv error
------ | ------ | ---------- | ------------- | --------------------- | --------
0 | xg_boost | **=> 1 <=** | -79.89 | -83.8, -75.98 | -76.92

Does that imply that the feature selector removed every but one feature?